python.py

```diff
@@ -5,4 +5,5 @@
         print (element)
     else:
         print("it is not a list.")
-        print("Add this line")
+        print("its okay")
```

Comment:
Why you thought that this is better than the old one  ?

Reply:
Because to reduce the time that the code has to fo through each loop. I think it way more better